src/lib/bootloader/config_dialog.rb

```diff
@@ -27,7 +27,8 @@ def run
     rescue ::Bootloader::NoRoot
       Yast::Report.Error(
         _("YaST cannot configure the bootloader because it failed to find the root file system.\n" \
-          "This usually happens when there are bind-mounted devices.")
+          "This usually happens when there are bind-mounted devices. Please, unmount your bind \n" \
+          "mounts first and try again. You can mount it back after configuring your bootloader.")
```

Comment:
Shouldn't it be "You can mount THEM back"? It's referring to "your bind mounts" (plural).

Reply:
Sure.